setup.cfg

```diff
@@ -1,3 +1,62 @@
+[metadata]
+package_name = gammapy
```

Comment:
@bsipocz - do you know why the `package_name` isn't picked up?

I think how it should work is that those `setup.cfg` infos should be picked up automatically when running `python setup.py` or `pip`, one doesn't have to parse and pass along to `setuptools.setup` in `setup.py`?

I get this:
```
$ pip install -e .
Obtaining file:///Users/deil/work/code/gammapy
Requirement already satisfied: numpy>=1.10 in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from UNKNOWN==0.13.dev327+gefa8ea767) (1.16.4)
Requirement already satisfied: astropy>=2.0 in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from UNKNOWN==0.13.dev327+gefa8ea767) (3.2.1)
Requirement already satisfied: scipy>=0.15 in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from UNKNOWN==0.13.dev327+gefa8ea767) (1.3.0)
Requirement already satisfied: regions>=0.4 in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from UNKNOWN==0.13.dev327+gefa8ea767) (0.4)
Requirement already satisfied: pyyaml in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from UNKNOWN==0.13.dev327+gefa8ea767) (5.1.1)
Requirement already satisfied: click in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from UNKNOWN==0.13.dev327+gefa8ea767) (7.0)
Requirement already satisfied: six in /Users/deil/software/anaconda3/envs/gammapy-dev/lib/python3.7/site-packages (from regions>=0.4->UNKNOWN==0.13.dev327+gefa8ea767) (1.12.0)
Installing collected packages: UNKNOWN
  Found existing installation: UNKNOWN 0.13.dev326+gf2a703b69.d20190714
    Uninstalling UNKNOWN-0.13.dev326+gf2a703b69.d20190714:
      Successfully uninstalled UNKNOWN-0.13.dev326+gf2a703b69.d20190714
  Running setup.py develop for UNKNOWN
Successfully installed UNKNOWN
```
What's wrong?

Reply:
Ah, looks like the key name here has to be `name`, not `package_name`.